src/main/scala/com/databricks/labs/mosaic/functions/MosaicContext.scala

```diff
@@ -609,6 +631,28 @@ class MosaicContext(indexSystem: IndexSystem, geometryAPI: GeometryAPI) extends
         def st_contains(geom1: Column, geom2: Column): Column = ColumnAdapter(ST_Contains(geom1.expr, geom2.expr, geometryAPI.name))
         def st_intersects(left: Column, right: Column): Column = ColumnAdapter(ST_Intersects(left.expr, right.expr, geometryAPI.name))
 
+        /** RasterAPI dependent functions */
+        def st_metadata(raster: Column, path: Column): Column =
+            ColumnAdapter(
+              ST_MetaData(raster.expr, path.expr)
+            )
+        def st_metadata(raster: Column): Column =
+            ColumnAdapter(
+              ST_MetaData(raster.expr, lit("").expr)
```

Comment:
NIT: I would move the `lit("").expr` as a default parameter if possible, in order to centralise the responsibility of the default values.

Reply:
This is no longer an issue. We dont need empty paths with new code.